spec/factories/factory_spec.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.factories.map(&:name).each do |factory_name|
+  describe "#{factory_name} factory" do
+
+    # Test each factory
+    it "is valid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.